diff --git a/lib/index.js b/lib/index.js
index e5b04f107f765db04265c7635d3ca1d86b146fb8..f8bd8488e2a624ef0a89db4c165e0510ffa9211c 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -668,7 +668,9 @@ Body.Promise = global.Promise;
  * Headers class offers convenient helpers
  */
 
-const invalidTokenRegex = /[^\^_`a-zA-Z\-0-9!#$%&'*+.|~]/;
+// fix http2 support for apollo server v3
+// ref: https://github.com/apollographql/apollo-server/issues/1533
+const invalidTokenRegex = /[^\^_`:a-zA-Z\-0-9!#$%&'*+.|~]/;
 const invalidHeaderCharRegex = /[^\t\x20-\x7e\x80-\xff]/;
 
 function validateName(name) {
