diff --git a/dist/index.es.js b/dist/index.es.js
index 68715495bdca59a7eac434bf88bacc8678ef1adb..22d573b2f2221723292c1a72674e31af6e6c1dfb 100644
--- a/dist/index.es.js
+++ b/dist/index.es.js
@@ -3765,6 +3765,7 @@ var MouseDown = function MouseDown(view, pos, event, flushed) {
   var ref = view.state;
   var selection = ref.selection;
   if (event.button == 0 &&
+      (!this.target || this.target.draggable !== false) &&
       targetNode.type.spec.draggable && targetNode.type.spec.selectable !== false ||
       selection instanceof NodeSelection && selection.from <= targetPos && selection.to > targetPos)
     { this.mightDrag = {node: targetNode,
