diff --git a/node_modules/@rails/actioncable/app/assets/javascripts/action_cable.js b/node_modules/@rails/actioncable/app/assets/javascripts/action_cable.js
index 8d14edb..172741b 100644
--- a/node_modules/@rails/actioncable/app/assets/javascripts/action_cable.js
+++ b/node_modules/@rails/actioncable/app/assets/javascripts/action_cable.js
@@ -148,7 +148,7 @@
     max: 30,
     multiplier: 5
   };
-  ConnectionMonitor.staleThreshold = 6;
+  ConnectionMonitor.staleThreshold = 60;
   var INTERNAL = {
     message_types: {
       welcome: "welcome",
diff --git a/node_modules/@rails/actioncable/src/connection_monitor.js b/node_modules/@rails/actioncable/src/connection_monitor.js
index 3bcee72..4f19965 100644
--- a/node_modules/@rails/actioncable/src/connection_monitor.js
+++ b/node_modules/@rails/actioncable/src/connection_monitor.js
@@ -121,6 +121,6 @@ ConnectionMonitor.pollInterval = {
   multiplier: 5
 }
 
-ConnectionMonitor.staleThreshold = 6 // Server::Connections::BEAT_INTERVAL * 2 (missed two pings)
+ConnectionMonitor.staleThreshold = 60 // Server::Connections::BEAT_INTERVAL * 2 (missed two pings)
 
 export default ConnectionMonitor
