diff --git a/node_modules/prosemirror-model/dist/index.es.js b/node_modules/prosemirror-model/dist/index.es.js
index b1626f4..50f56de 100644
--- a/node_modules/prosemirror-model/dist/index.es.js
+++ b/node_modules/prosemirror-model/dist/index.es.js
@@ -1,4 +1,5 @@
 import OrderedMap from 'orderedmap';
+import { v4 as uuid } from 'uuid';
 
 function findDiffStart(a, b, pos) {
   for (var i = 0;; i++) {
@@ -1014,7 +1015,7 @@ var Node = function Node(type, attrs, content, marks) {
   // An object mapping attribute names to values. The kind of
   // attributes allowed and required are
   // [determined](#model.NodeSpec.attrs) by the node type.
-  this.attrs = attrs;
+  this.attrs = {uuid: uuid(), sort: 0, ...attrs};
 
   // :: Fragment
   // A container holding the node's children.
@@ -1865,6 +1866,13 @@ function computeAttrs(attrs, value) {
     }
     built[name] = given;
   }
+
+  if (value?.uuid) {
+    built.uuid = value.uuid
+  }
+  if (value?.sort) {
+    built.sort = value.sort
+  }
   return built
 }
 
